integration-tests/smoke/keeper_test.go

```diff
@@ -34,10 +34,9 @@ var _ = Describe("Keeper suite @keeper", func() {
 			env, err = environment.DeployOrLoadEnvironment(
 				environment.NewChainlinkConfig(
 					environment.ChainlinkReplicas(6, config.ChainlinkVals()),
-					"chainlink-keeper-core-ci",
```

Comment:
@infiloop2 any reason why the suffix `-core-ci` to the namespaces were removed across tests? I believe this was done to easily identify ci tests running in the cluster.

Reply:
I saw this was changed in testing framework repo: https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/suite/smoke/keeper_test.go#L36 so copied that over with the other changes. Should i add it back?

Reply:
I think it should remain as before. This change was introduced [here](https://github.com/smartcontractkit/chainlink/pull/6446/files#diff-0b3c9e31377793dfbe89de3c72462a6146004c64c3fcce0ce127b26cce86a018) , which is after the [changes](https://github.com/smartcontractkit/chainlink-testing-framework/pull/257/files#diff-c60c919044f6f193388142f9ed6754e445b6699a6064303a55630862c1f8ba7c) to the chainlink-testing-framework repo.

cc: @kalverra

Reply:
Done